grpc/src/test/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandlerBuilderTest.java

```diff
@@ -0,0 +1,201 @@
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessageContaining(
+                        "Cannot register the collection of messages because a custom JSON marshaller has " +
+                        "already been set. Use the custom marshaller to register custom messages.");
+
+        assertThatThrownBy(
+                () -> UnframedGrpcErrorHandler.builder()
+                                              .jsonMarshaller(
+                                                      UnframedGrpcErrorHandlers.ERROR_DETAILS_MARSHALLER)
+                                              .registerMarshalledMessageTypes(
+                                                      ImmutableList.of(InternalError.class)))
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessageContaining(
+                        "Cannot register the collection of messageTypes because a custom JSON marshaller has " +
+                        "already been set. Use the custom marshaller to register custom message types.");
+
+        assertThatThrownBy(
+                () -> UnframedGrpcErrorHandler.builder()
+                                              .registerMarshalledMessages(
+                                                      InternalError.newBuilder().build())
+                                              .jsonMarshaller(
+                                                      UnframedGrpcErrorHandlers.ERROR_DETAILS_MARSHALLER))
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessageContaining(
+                        "Cannot set a custom JSON marshaller because one or more Message or " +
+                        "MessageType instances have already been registered. To set a custom marshaller, " +
+                        "ensure that no Message or MessageType registrations have been made before " +
+                        "calling this method.");
+    }
+
+    @Test
+    void buildWithoutOptions() {
+        final UnframedGrpcErrorHandler unframedGrpcErrorHandler = UnframedGrpcErrorHandler.builder().build();
+        final ServiceRequestContext ctx = ServiceRequestContext.of(
+                HttpRequest.of(RequestHeaders.of(HttpMethod.GET, "/test")));
+        final AggregatedHttpResponse jsonResponse =
+                AggregatedHttpResponse.of(HttpStatus.INTERNAL_SERVER_ERROR,
+                                          MediaType.JSON_UTF_8,
+                                          "{\"message\":\"Internal Server Error\"}");
+        final HttpResponse httpResponseJson = unframedGrpcErrorHandler.handle(ctx, Status.INTERNAL,
+                                                                              jsonResponse);
+        final AggregatedHttpResponse aggregatedHttpResponse = httpResponseJson.aggregate().join();
+        assertThat(requireNonNull(aggregatedHttpResponse.headers().contentType())
```

Comment:
`requireNonNull()` is redundant because `assertThat()` will handle a `null` parameter rather than throwing a NPE.